### ISSUE TITLE
The code is displayed directly in the email template

### ISSUE THREAD

**Issue by dima-ogrenich** [putyourlightson/craft-campaign#347]
Steps to reproduce the issue:
- send a test newsletter;

The JS code is displayed directly in the email template:

<img width="527" alt="image" src="https://user-images.githubusercontent.com/66615356/204738014-c209df6a-44ab-4b2c-92c2-916beb44e03c.png">

PHP version	8.0.25
OS version	Linux 4.18.0-348.20.1.lve.1.el7h.x86_64
Database driver & version	MySQL 10.6.11
Image driver & version	GD 8.0.25
Craft edition & version	Craft Pro 4.3.3
Yii version	2.0.46
Twig version	v3.4.3
Guzzle version	7.5.0

Campaign	2.2.3

**Comment by bencroker** [comment]
Where exactly are you sending the test from, a campaign or a sendout? Please clear all caches in Craft to ensure this isn't coming from stale content.

**Comment by dima-ogrenich** [comment]
I cleared all cache.
<img width="719" alt="image" src="https://user-images.githubusercontent.com/66615356/204819653-5489cadd-c6d4-461d-9832-86e935589644.png">

**Comment by bencroker** [comment]
I'm not sure where this code is coming from – it's possible that a module/plugin is injecting it when it shouldn't be. Can you test this with a very simple HTML template, for example:

```twig
<!doctype html>
<html>
<head>
    <title>{{ campaign.title }}</title>
</head>
<body>
    Hello, {{ contact.email }}!
</body>
</html>
```

**Comment by dima-ogrenich** [comment]
Got the same result:

<img width="537" alt="image" src="https://user-images.githubusercontent.com/66615356/205020413-f2a67372-ffa0-43ac-a577-23b9a3a446fb.png">

**Comment by bencroker** [comment]
Thanks for checking. It looks like your email client is not respecting `<script>` tags, but either way these should not be included in the template, I'll investigate further.

**Comment by bencroker** [comment]
I've fixed this for the upcoming 2.3.0 release. If you'd like to help test then you can change the `putyourlightson/craft-campaign` requirement in your project's `composer.json` file to "`dev-develop as 2.3.0`" and run `composer update`.

**Comment by wmdhosting** [comment]
This works.. ok now on my side too

**Comment by bencroker** [comment]
Great! Released in version 2.3.0.
